### PR TITLE
#138 new zoom-control example

### PR DESCRIPTION
**Pull request by OpenSteveMap**

### CODE REVIEW

File: examples/zoom-control-directive/index.d.ts [hunk 1]
```diff
@@ -1,18 +1,8 @@
...
-    position: ControlPosition;
-    zoomInText: string;
-    zoomInTitle: string;
-    zoomOutText: string;
-    zoomOutTitle: string;
-    private mapComponent;
-    constructor();
-    handlePositionEvent(event: Event): void;
-    ngAfterViewInit(): void;
+import { ExampleAppComponentBlueprint, IExampleProperties } from '../app-component-blueprint';
```
Comment (atd-schubert) [inline examples/zoom-control-directive/index.d.ts]: we have to kick out the d.ts files from our repository. They are generated files and do not have to be placed into the repo... see #158

File: examples/zoom-control-directive/index.ts [hunk 2]
```diff
@@ -2,133 +2,100 @@
...
+
+        [zoomInText]="getInputPropertyByName('zoomInText').value"
+        [zoomInTitle]="getInputPropertyByName('zoomInTitle').value"
+        [zoomOutText]="getInputPropertyByName('zoomOutText').value"
+        [zoomOutTitle]="getInputPropertyByName('zoomOutTitle').value"
+        
+        [position]="getDuplexPropertyByName('position').value"
+        [(zIndex)]="getDuplexPropertyByName('zIndex').value"
+        [(display)]="getDuplexPropertyByName('display').value"
+        [opacity]="getDuplexPropertyByName('opacity').value"
```
Comment (atd-schubert) [inline examples/zoom-control-directive/index.ts]: Bind duplex properties in duplex mode

Here a little mule-bridge: `[(duplex)]`, `[input]`, `(output)`

Comment (atd-schubert) [reply examples/zoom-control-directive/index.ts]: That means [**(** position **)**] and [**(** opacity **)**]

**Review by atd-schubert — CHANGES_REQUESTED** [review]

**Review by OpenSteveMap — COMMENTED** [review]
ok

**Review by atd-schubert — APPROVED** [review]
Correct. Opacity is not duplex in this directive! So great again ;)

**Pull request merged**